great-idea/css/index.css

```diff
@@ -63,4 +63,134 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+body {
+	box-sizing: border-box;
+
+	margin: 0 auto;
+}
+nav {
+	display: flex;
+	flex-flow: row wrap;
+	justify-content: space-evenly;
+	background-color: #59baf7;
+	position:fixed;
+	top:0;
+	width:100%;
+	margin-bottom: 50px;
+
+}
+nav a {
+	margin: 12px 12px;
+	text-decoration: none;
+	color:black;
+	font-size:14px;
+}
+.container {
+	display: flex;
+	align-items: center;
+	border-bottom: 1px solid gray;
+	padding: 50px 0;
+	margin-bottom: 30px;
+}
+.phrase {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+h1 {
+	width:50%;
+	text-align: center;
+	font-size: 48px;
+}
+.button {
+	border: 1px dashed black;
+	width: 30%;
+	height: 30%;
+}
+.button p {
+	text-align: center;
+}
+.imageOne{
+	display:flex;
+	height: 75%;
+	width: 75%;
+	justify-content: center;
+}
+.imageCode {
+	margin-top: 40px;
+	height: 45%;
+	width: 45%;
+}
+.foldOne {
+	display: flex;
+	padding: 10px 0;
+	justify-content: space-between;
+	margin-bottom: 20px;
+}
+.features {
+	padding-left: 17px;
+	width: 40%;
```

Comment:
i think these two could be a little wider than 40% to line up with the design file a little better